src/main/java/org/somerville/swag/data/entity/state/Guest.java

```diff
@@ -45,7 +63,7 @@ public void removeProductFromBasket() {
 
     @Override
     public void purchaseItems() {
-
+        // TODO: Display error that you need to be logged in to checkout
```

Comment:
These TODO are really useful, it is clear what needs to be worked on next

Reply:
Thank you!